### PR TITLE
Update description for services

### PR DESCRIPTION
**Pull request by fangfy**

### CODE REVIEW

**Review by fangfy — COMMENTED** [review]
For the GeoMAD page, can we change top header to "Service overview"? instead of product overview.
Also, it would be nice to add something brief in the background that GeoMAD can be provided at different time intervals. Annual is good for tracking changes year to year and more frequent GeoMAD can detect seasonal changes etc. Perhaps this can be combined with top paragraph in specifications.

File: data_specs/GeoMAD_specs.ipynb [hunk 1]
```diff
@@ -65,14 +65,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The GeoMAD dataset is annual (one value per pixel per band per calendar year) and spans continental Africa. GeoMAD metadata can also be viewed on Digital Earth Africa's [Metadata Explorer](https://explorer.digitalearth.africa/products/ga_s2_gm)."
+    "The GeoMAD service spans all of Africa and is currently produced in two timeframes:\n",
+    "\n",
+    "* Annual: one value per pixel per band per calendar year\n",
```
[the remark this answers is not in the document]
Comment (eefaye) [reply data_specs/GeoMAD_specs.ipynb:74]: Hmm it says "per band", is that not accurate? I agree the wording is a bit confusing, will try and streamline it

[the remark this answers is not in the document]
Comment (eefaye) [reply data_specs/GeoMAD_specs.ipynb]: Text has been updated.

**Comment by eefaye** [comment]
@fangfy Just planning to add some semiannual images, but text has otherwise been reviewed. Looking to do the images once the Terria issues have been resolved. Confirmation of text + any more changes would be great

**Review by eefaye — APPROVED** [review]

**Pull request merged**